widget/ui/src/components/Toast/Toast.tsx

```diff
@@ -14,23 +14,26 @@ import {
   ToastContentContainer,
 } from './Toast.styles';
 
-const DEFAULT_HIDE_DURATION = 3_000;
-
 export const Toast = (props: ToastProps) => {
-  const { id, horizontal, autoHideDuration = DEFAULT_HIDE_DURATION } = props;
+  const { id, autoHideDuration, onClose } = props;
```

Comment:
I didn't understand why we have removed default value here.

Reply:
If it has autoHideDuration it is removed if not the user should handle it by `removeToast` manually. in dapp some toasts have` autoHide={false}`

we can also consider having 2 props: `autoHide` & `autoHideDuration`. but currently, user can set it by `autoHideDuration`